### ISSUE TITLE
Minor addition to documentation: #hide at end of line

### ISSUE THREAD

**Issue by moustachio-belvedere** [fredrikekre/Literate.jl#129]
In section 2.2 of the documentation (https://fredrikekre.github.io/Literate.jl/v2/fileformat/#Filtering-Lines) it says that:

> The `#src` token can also be placed at the end of a line. This is to make it possible to have code lines exclusive to the source code, and not just comment lines.

However, I noticed that the `#hide` token can also be placed at the end of the line, and this particularly useful, for example in relation to https://github.com/fredrikekre/Literate.jl/issues/36 if you don't want the `gcf()` command to show up in the docs.

I'm happy to add a PR with a small pull request to add this to the documentation.

I'm not sure what the best workflow is as I have another PR pending on my fork. Let me know what you think is best. Happy for anyone else to make the change if that is easier.

**Comment by moustachio-belvedere** [comment]
Apologies, I didn't read the paragraphs above it carefully enough where it says:

> Lines starting or ending with one of these tokens are filtered out in the preprocessing step. In addition, for markdown output, lines ending with #hide are filtered out similar to Documenter.jl.

Closing for now.
